pkg/virtctl/memorydump/memorydump.go

```diff
@@ -169,6 +169,8 @@ func generatePVC(size *resource.Quantity, claimName, namespace, storageClass, ac
 
 	if accessMode == string(k8sv1.ReadOnlyMany) {
 		return nil, fmt.Errorf("cannot dump memory to a readonly pvc, use either ReadWriteOnce or ReadWriteMany if supported")
+	} else if accessMode != "" && accessMode != string(k8sv1.ReadWriteOnce) && accessMode != string(k8sv1.ReadWriteMany) {
```

Comment:
I'm not quite sure I understand this condition. Do you want to avoid the ReadWriteOncePod? It cannot be ReadOnlyMany as you have already check it

Reply:
What about ReadWriteOncePod? I think it should be valid

Reply:
you can see in the bug that the pvc webhook returns it supports only: "PersistentVolumeClaim "memoryvolume" is invalid: spec.accessModes: Unsupported value: "RWX": supported values: "ReadOnlyMany", "ReadWriteMany", "ReadWriteOnce""
Then for memory dump I guess it only supports "ReadWriteMany", "ReadWriteOnce". I can test "ReadWriteOncePod" and see what happens without my validation, but thats what I understand from the pvc error.

Reply:
I checked it and it is as I said, the pvc webhook doesnt support `ReadWriteOncePod`

Reply:
I think we need a new issue to update Datavolume Validator for new kubernetes

Reply:
Found it, it already exists: https://github.com/kubevirt/containerized-data-importer/issues/2365 
So I suppose currently the code is OK, but it needs a note/comment that it can be extended to support ReadWriteOncePod when CDI issue 2365 is implemented.